dotCMS/src/main/java/com/dotmarketing/common/util/SQLUtil.java

```diff
@@ -310,4 +316,134 @@ private static boolean isValidSQLCharacter (final char c) {
 		return Character.isLetterOrDigit(c) || '-' == c || '_' == c;
 	} // isValidSQLCharacter.
 
+
+	//H2 Upsert Example:
+	//MERGE INTO table (columns) KEY (conditionalColumn) VALUES (values)
+	private final static String H2_UPSERT_QUERY =
+			"MERGE INTO %s (%s) KEY (%s) VALUES (%s) ";
+
+	// Postgres Upsert Example:
+	// INSERT INTO table (columns) VALUES (values)
+	// ON CONFLICT (conditionalColumn) DO UPDATE SET column1=value1, column2=value2, etc...
+	private final static String POSTGRES_UPSERT_QUERY =
+			"INSERT INTO %s (%s) "
+			+ "VALUES (%s) ON CONFLICT (%s) "
+			+ "DO UPDATE SET %s";
+
+	// MySQL Upsert Example:
+	// INSERT INTO table (columns) VALUES (values)
+	// ON DUPLICATE KEY UPDATE column1=value1, column2=value2, etc...
+	private final static String MYSQL_UPSERT_QUERY =
+			"INSERT INTO %s (%s) "
+			+ "VALUES (%s) ON DUPLICATE KEY "
+			+ "UPDATE %s";
+
+	// MSSQL Server Upsert Example:
+	// MERGE table WITH (HOLDLOCK) AS [Target] USING
+	// (SELECT conditionalValue AS conditionalColumn) AS [Source] ON [Target].conditionalColumn = [Source].conditionalColumn
+	// WHEN MATCHED THEN UPDATE SET column1=value1, column2=value2, etc...
+	// WHEN NOT MATCHED THEN INSERT (columns) VALUES (values);
+	private final static String MSSQL_UPSERT_QUERY =
+			"MERGE %s WITH (HOLDLOCK) AS [Target] USING "
+			+ "(SELECT %s AS %s) AS [Source] ON [Target].%s = [Source].%s "
+			+ "WHEN MATCHED THEN "
+			+ "  UPDATE SET %s "
+			+ "WHEN NOT MATCHED THEN "
+			+ "  INSERT (%s) "
+			+ "  VALUES (%s);";
+
+	// Oracle Upsert Example:
+	// INSERT INTO table Target USING
+	// (SELECT conditionalValue conditionalColumn FROM dual) Source ON (Target.conditionalColumn = Source.conditionalColumn)
+	// WHEN MATCHED THEN UPDATE SET column1=value1, column2=value2, etc...
+	// WHEN NOT MATCHED THEN INSERT (columns) VALUES (values)
+	private final static String ORACLE_UPSERT_QUERY =

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)